src/Parser/RestoDruid/Modules/Features/StatWeights.js

```diff
@@ -0,0 +1,316 @@
+import React from 'react';
+import SPELLS from 'common/SPELLS';
+import Module from 'Parser/Core/Module';
+import HIT_TYPES from 'Parser/Core/HIT_TYPES';
+import { formatNumber } from 'common/format';
+import Combatants from 'Parser/Core/Modules/Combatants';
+import HealingValue from 'Parser/Core/Modules/HealingValue';
+import CritEffectBonus from 'Parser/Core/Modules/Helpers/CritEffectBonus';
+
+import { getSpellInfo } from '../../SpellInfo';
+
+const DEBUG = false;
+
+const ARMOR_INT_MULTIPLIER = 1.05; // 5% int bonus from wearing all leather means each new point of int worth 1.05 vs character sheet int
+
+/*
+ * Resto Druid Stat Weights Methodology
+ *
+ * Approach -
+ * This module generates the players stat weights using the actual logged events. We keep a listing of all the player's healing spells
+ * along with which stats those spells scales with, and for each stat a heal scales with we do some simple math to find out
+ * how much more it would have healed if the player had one more of that stat. We compare the total healing increases
```

Comment:
`for each stat a heal scales with we do some simple math to find out how much more it would have healed if the player had one more of that stat`

If I understand the code correctly I think this isn't exactly right. It's actually `for each stat a heal scales with we do some simple math to find out how much healing one of that stat contributed`, no? Seeing as you seem to calculate the actual contribution of the last value of each stat, not the gain from adding 1 stat. It's no big deal though.

Reply:
I think of it as the gain from one stat, but it's functionally the same thing.